core/download.ts

```diff
@@ -40,6 +40,12 @@ function detectType(text: string): string | undefined {
   }
 }
 
+function detectEncodeType(response: Partial<Response>): string {
```

Comment:
```suggestion
function parseEncodeType(response: Partial<Response>): string {
```

Since we do not detect it (by byte orders, etc).

Reply:
updated